### PR TITLE
add python solution cses-1634

### PR DESCRIPTION
**Pull request by 876pol**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Review by brad182 — CHANGES_REQUESTED** [review]
I know that the code is relatively short, but could you add a few comments?

File: solutions/cses-1634.mdx [hunk 1]
```diff
@@ -153,4 +153,26 @@ public class cses1634 {
 ```
 </JavaSection>
 
+<PySection>
+```python
+INF = 1000000000
```
Comment (SansPapyrus683) [inline solutions/cses-1634.mdx:158]: or just `float('inf')`

Comment (876pol) [reply solutions/cses-1634.mdx:158]: putting that makes the code tle for some reason 💀

**Comment by 876pol** [comment]
> I know that the code is relatively short, but could you add a few comments?

added

File: solutions/cses-1634.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/cses-1634.mdx:158]: add a comment saying that then

Comment (876pol) [reply solutions/cses-1634.mdx]: ok just added

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by 876pol** [comment]
can this maybe be merged?

**Pull request merged**